WRD Guide Version 2 (#1853)
telerik/reporting-docs: user-guide/customizing-report-items/customizing-report-items-overview.md

---
title: Overview
page_title: Web Report Designer - Customizing Report Items Overview
description: Learn how to customize the visuals of your report using the fine-grained, built-in styling model, similar to the Cascading Style Sheets (CSS) model.
slug: web-report-designer-customizing-report-items
tags: web, report, design, customize, style, condition, formatting, properties, area 
published: True
position: 0
---
<style>
img[alt$="><"] {
  border: 1px solid lightgrey;
}
</style>

> # Customizing Report Items - Overview

Reviewer: ```suggestion
# Customizing Report Items: Overview
```

> A well-styled report looks professional and polished. Styling report items serves both, functional and aesthetic, purposes. Visual consistency (e.g., using a theme or style rules) improves user experience and brand alignment. It allows you to improve readability and clarity in your reports. Applying different colors or fonts distinguishes separate sections or highlight key information.

Reviewer: ```suggestion
A well-styled report looks professional, clear, and is easy to read. Styling report items serves both functional and aesthetic, purposes. Visual consistency (for example, using the same theme or style rules) improves user experience and supports brand alignment. Applying different colors or fonts lets you highlight specific sections or key information.
```

To customize or style report items in the Telerik Web Report Designer, you can use its built-in CSS-like styling model and the Properties panel.

## Properties Area - What Is It For

The **Properties Area** lets you control how each report item looks and behaves. Whether you are working with a TextBox, Table, Chart, or Image, this panel gives you access to all the relevant settings in one place:

> * **Style your items**: Change fonts, colors, borders, background, padding, and more.

Reviewer: ```suggestion
* **Style items**&mdash;Change fonts, colors, borders, backgrounds, paddings, and more.
```

* **Set data bindings**: Connect report items to fields or expressions from your data source.
* **Adjust layout**: Modify size, position, alignment, and visibility.
* **Apply conditional formatting**: Dynamically change appearance based on data values.

> * **Configure behavior**: Set actions like navigation, visibility rules, or interactivity.

Reviewer: ```suggestion
* **Configure behavior**&mdash;Set actions like navigation, visibility rules, or interactivity.
```
